Documentation/documentation-styles.css

```diff
@@ -0,0 +1,114 @@
+/* Import */
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans&display=swap');
+
+/* Root */
+:root {
+    --primary-color-d: #2a9d8f;
+    --primary-dark-d: #023047;
+    --primary-grey-d: #eaeaea;
+    --primary-light-color-d: #e63946;
+    --white-d: #ffffff;
+}
+
+
+html {
+    scroll-behavior: smooth;
+}
+/* Body */
+body {
+    font-family: [Open Sans], sans-serif;
+    padding: 0;
+    margin: 0;
+}
+
+/* Navbar */
+.navbar-d {
+    padding: 1rem;
+    color: var(--primary-dark-d);
```

Comment:
can't figure out what the d stands for

Reply:
The "d" is added for styling the documentation, so the classes doesn't clash with component library.